src/SFA.DAS.ApprenticeFeedback.Data/Repository/ApprenticeFeedbackRepository.cs

```diff
@@ -59,10 +60,23 @@ public async Task<ApprenticeFeedbackTarget> GetApprenticeFeedbackTarget(Guid app
         => await _dbContext.ApprenticeFeedbackTargets.
             FirstOrDefaultAsync(aft => aft.ApprenticeId == apprenticeId && aft.ApprenticeshipId == commitmentsApprenticeshipid);
 
+        public async Task<ApprenticeFeedbackTarget> GetApprenticeFeedbackTargetById(Guid apprenticeId)
```

Comment:
I think this method can be changed to use the ApprenticeFeedbackTargetId and also be single or default when it's changed